packages/api/src/account/validators.ts

```diff
@@ -19,3 +19,35 @@ export function validateUpdateTagsRequest(
     throw validation.error;
   }
 }
+
+export function validateLeaveArchiveParams(
+  data: unknown
+): asserts data is { archiveId: string } {
+  const validation = Joi.object()
+    .keys({
+      archiveId: Joi.string()
+        .regex(/^[1-9]\d+$/)
```

Comment:
I would omit this check; while it's true that presently a string that doesn't look like a positive integer couldn't be an `archiveId`, we'd eventually like all our IDs to be UUIDs instead, so I don't want to add code that prohibits that.

Reply:
i decided to allow both numeric and uuid values, to have some sort of check in place.